libs/SMTPResponse/include/SMTPResponse.h

```diff
@@ -0,0 +1,37 @@
+#pragma once 
+
+#include <string>
+#include <regex>
+
+#include "StatusType.h"
+
+namespace ISXR
```

Comment:
Probably, it could be better make namespace name more self-explanatory.

Reply:
Agree with Viacheslav